docs/src/whatsnew/latest.rst

```diff
@@ -50,7 +55,10 @@ This document explains the changes made to Iris for this release
 🚀 Performance Enhancements
 ===========================
 
-#. N/A
+#. Note that due to the new ``date_microseconds`` :class:`~iris.Future` flag,
+   the time coordinate categorisation speedup introduced in
+   :doc:`/whatsnew/3.11` will only be available when
+   ``iris.FUTURE.date_microseconds == True``.
```

Comment:
@bouweandela please take note.

I spent about an hour trying to work out a way to use cftime's vectorisation when rounding a large number of time units, without success.

So to access your performance improvements you will need to activate this future flag as described.